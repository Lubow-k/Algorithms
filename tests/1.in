1234 5678
1 0
962105 89023
417005 702495
608036 215942
584799 722305
159527 144098
323641 332706
141975 647799
51634 472990
512748 244376
90691 114920
762814 629288
192752 476358
676571 737342
161125 708475
428945 388645
391513 577938
865777 580463
907845 843327
309263 667512
501667 810466
537580 493434
46721 606203
130207 16633
387182 487741
460018 486989
431291 650042
884279 128427
411745 901889
246557 14951
785396 412284
713881 735017
942252 219462
717578 125231
104719 461687
987949 364279
159393 252278
994902 479913
880576 445802
551451 335583
907403 274031
388053 817136
324824 497709
711621 622730
738334 27345
823279 703165
657621 858875
921845 331441
900366 669914