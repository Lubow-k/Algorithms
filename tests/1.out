7006652
0
85649473415
292943927475
131300509912
422403241695
22987521646
107677302546
91971263025
24422365660
125303305248
10422209720
480029696432
91818957216
498864214282
114153034375
166707329525
226270240194
502551514751
765610200315
206436763656
406584046822
265260249720
28322410363
2165733031
188844535862
224023705802
280357264222
113565299133
371348286305
3686273707
323806204464
524714670977
206788508424
89863010518
48347400953
359889073771
40211347254
477466403526
392562541952
185057580933
248656551493
317092076208
161667828216
443147745330
20189743230
578900978035
564814236375
305537228645
603167788524